src/Symfony/Component/Notifier/Transport/Transports.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Transport;
+
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @experimental in 5.0
+ */
+class Transports implements TransportInterface
+{
+    private $transports;
+    private $default;
+
+    /**
+     * @param TransportInterface[] $transports
+     */
+    public function __construct(iterable $transports)
+    {
+        $this->transports = [];
+        foreach ($transports as $name => $transport) {
+            if (null === $this->default) {
+                $this->default = $transport;
+            }
+            $this->transports[$name] = $transport;
+        }
+    }
+
+    public function __toString(): string
+    {
+        return 'all';
+    }
+
+    public function supports(MessageInterface $message): bool
+    {
+        foreach ($this->transports as $transport) {
+            if ($transport->supports($message)) {
```

Comment:
is this enough to decide? looking at the implementation in send, that's a bit more complex, isn't it?